android/app/src/main/java/com/mattermost/rnbeta/MainApplication.java

```diff
@@ -87,78 +54,34 @@
 
   private Bundle mManagedConfig = null;
 
-  @Override
-  protected ReactGateway createReactGateway() {
-    ReactNativeHost host = new NavigationReactNativeHost(this, isDebug(), createAdditionalReactPackages()) {
-      @Override
-      protected String getJSMainModuleName() {
-        return "index";
-      }
-    };
-    return new ReactGateway(this, isDebug(), host);
-  }
-
-  @Override
-  public boolean isDebug() {
-    return BuildConfig.DEBUG;
-  }
+private final ReactNativeHost mReactNativeHost =
+  new ReactNativeHost(this) {
+    @Override
+    public boolean getUseDeveloperSupport() {
+      return BuildConfig.DEBUG;
+    }
 
-  @NonNull
-  @Override
-  public List<ReactPackage> createAdditionalReactPackages() {
-    // Add the packages you require here.
-    // No need to add RnnPackage and MainReactPackage
-    return Arrays.<ReactPackage>asList(
-            new TurboReactPackage() {
+    @Override
+    protected List<ReactPackage> getPackages() {
+      @SuppressWarnings("UnnecessaryLocalVariable")
+      List<ReactPackage> packages = new PackageList(this).getPackages();
+      // Packages that cannot be autolinked yet can be added manually here, for example:
+      // packages.add(new MyReactNativePackage());
+      packages.add(new RNNotificationsPackage(MainApplication.this));
```

Comment:
Would passing `instance` be equivalent?

Reply:
I guess it would, but I don't want to deviate from the docs